api/leaderboard.go

```diff
@@ -406,6 +406,55 @@ func GetAroundMemberHandler(app *App) func(c echo.Context) error {
 	}
 }
 
+// GetAroundRankHandler retrieves a list of member score and rank centered in the given member
```

Comment:
// GetAroundScoreHandler retrieves a list of member score and rank centered in a given score